iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -557,7 +559,6 @@ protected override void Dispose(bool disposing)
                 if (disposing)
                 {
                     _closed = true;
-                    OnTransportClosedGracefully();
```

Comment:
This is called during `base.Dispose()`

Reply:
I am concerned about the ordering of the transport closure calls - previously we would call `OnTransportClosedGracefully` only if `disposing` was true, but now we will call it anytime Dispose() is called.

Reply:
I don't think that is the case. We call base.Dispose(disposing) so the order for the transport closure is intact.